README.md

```diff
@@ -24,6 +24,7 @@ Summary
 * [Plugins](content/docs/plugins.md)
 * [Preferences](content/docs/preferences.md)
 * [Themes](content/docs/themes.md)
+* [Plugin Development: Messages and Notifications](content/docs/messages-notifications.md)
```

Comment:
1. The short title "Plugin Development" is enough IMO
2. "Plugins" is better to be followed by "Plugin Development" IMO